src/components/Card.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import {ImageUrl} from '../services';
+import {cryptoData} from '../model/crypto.model';
+import styled from 'styled-components/native';
+import ArrowUp from 'react-native-vector-icons/Feather';
+import ArrowDown from 'react-native-vector-icons/Feather';
+const Card = (props: cryptoData) => {
+  const formatter = new Intl.NumberFormat('en-US', {
+    style: 'currency',
+    currency: 'USD',
+    minimumFractionDigits: 0,
+  });
+  const roundNumber =
+    props.metrics.market_data.percent_change_btc_last_24_hours.toFixed(2);
+  const roundNegative = Number(roundNumber) < 0;
+  return (
+    <CryptoCard>
+      <Logo>
+        <CryptoImage
+          source={{
+            uri: `${ImageUrl}${props.id}/64.png?v=`,
+          }}
+        />
+        <TextLogo>
+          <FirstText>{props.name}</FirstText>
+          <SecondText>{props.symbol}</SecondText>
+        </TextLogo>
+      </Logo>
+      <DataCrypto>
+        <FirstData>
+          <FirstText>
+            {formatter.format(props.metrics.market_data.price_usd)}
+          </FirstText>
+        </FirstData>
+        {roundNegative ? (
```

Comment:
Rendundant code, you can optimize this validation:
TextVariantDown and TextVariantUp should be the same component, and you can send props to a styled component 

https://styled-components.com/docs/basics#adapting-based-on-props